app/assets/javascripts/app/collections/notifications.js

```diff
@@ -0,0 +1,109 @@
+app.collections.Notifications = Backbone.Collection.extend({
+  model: app.models.Notification,
+  // URL parameter
+  /* eslint-disable camelcase */
+  url: Routes.notifications({per_page: 10, page: 1}),
+  /* eslint-enable camelcase */
+  page: 2,
+  perPage: 5,
+  unreadCount: 0,
+  unreadCountByType: {},
+
+  initialize: function() {
+    this.pollNotifications();
```

Comment:
I can, even if d-fr is not a big pod like frama or jd so I would tend to pick the careful way here and choose 1 or 2 minutes at the moment.